packages/toolpad-app/src/toolpad/AppEditor/PageEditor/RenderPanel/NodeDropArea.tsx

```diff
@@ -51,65 +51,63 @@ const StyledNodeDropArea = styled('div', {
     position: 'absolute',
     [`&.${dropAreaHighlightClasses.highlightedTop}`]: {
       '&:after': {
-        backgroundColor: '#44EB2D',
+        backgroundColor: blue[500],
```

Comment:
Seeing `blue[500]` used in a lot of places. If there's a shared purpose to these usages, perhaps it makes sense to assign it to a meaningfully named variable? That makes is self-documenting.

Reply:
I had misunderstood how we use the theme - it turns out that this color was in `theme.palette.primary` all along.
So I changed all these `blue[500]` to use the theme and also made the objects with colors not be exported directly from the theme file, so that we're always forced to use the theme.